src/irr/asset/IAssetManager.cpp

```diff
@@ -39,6 +39,10 @@
 #include "irr/asset/CImageLoaderTGA.h"
```

Comment:
you might want to convert all `_IRR_COMPILE_WITH` into CMake options that then go to a configure file

Reply:
sure, I will deal with it after having openexr loader done